### PR TITLE
Update outdated loki snapshots

### PR DESCRIPTION
**Pull request by WiNloSt**
### Description

Reported in https://metaboat.slack.com/archives/C5XHN8GLW/p1723834655159299. After my investigation, I concluded that these are actual failure since the padding on the failed tests in the report were different than other snapshots.

I ran all the tests locally and got all the same 4 failed tests as in [this check](https://github.com/metabase/metabase/actions/runs/10446943123). ~So I just updated these 4 snapshots.~ The line chart still flakes after some more runs, so I reverted it for now, I only update the embedding snapshots instead.

### How to verify

Loki CI check doesn't fail. (Welp, it still fails on the line chart, but at least all other failures are less flaky now)

### Checklist

- [x] Tests have been added/updated to cover changes in this PR (update snapshots)

### CODE REVIEW

File: .loki/reference/chrome_laptop_viz_LineChart_Embedding_Huge_Font.png
Comment (WiNloSt) [inline .loki/reference/chrome_laptop_viz_LineChart_Embedding_Huge_Font.png:1]: This one was still failing, so I used the snapshot generated on CI instead. The rest 3 snapshots were generated on my machine.

File: frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx
Comment (uladzimirdev) [inline frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx:122]: afaik `ExplicitSize` has different strategies of delaying re-renderings (debounce, throttle etc) and for cypress it's completely disabled. should we disabled any delay for storybook/loki as well?

please check https://github.com/metabase/metabase/blob/master/frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx#L122-L129

Comment (WiNloSt) [reply frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx:122]: It's already disabled here
https://github.com/metabase/metabase/blob/23725b4e9dcde164294b34966d3246ea3a909d08/frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx#L77-L79

Comment (WiNloSt) [reply frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx:122]: Even with that disabled we still need to wait for this on the CI from my testing unfortunately.

Comment (uladzimirdev) [reply frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx:122]: okay, thanks for explanation, but anyway, having a hardcoded value is not a good solution here. Is it possible to have some promise that will be resolved only when resizing is finished?

Comment (WiNloSt) [reply frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx:122]: If we could do that then it'd be great. I couldn't figure out how to get that info/create that kind of promise though.

Comment (WiNloSt) [inline frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx:122]: This doesn't take 50% more time for these tests, I ran tests locally and it increases the test suite time from 4-10% or (5 seconds at most on my Macbook pro)

**Comment by heypoom** [comment]
I think one of them has failed, let me update the branch so that it re-runs the Loki checks:

```
FAIL chrome.docker/chrome.laptop/viz/LineChart/Embedding Huge Font: Screenshot differs from reference, see .loki/difference/chrome_laptop_viz_LineChart_Embedding_Huge_Font.png
```

File: frontend/src/metabase/visualizations/visualizations/LineChart/LineChart.stories.tsx
Comment (heypoom) [inline frontend/src/metabase/visualizations/visualizations/LineChart/LineChart.stories.tsx:72]: It's for the SDK's themed variations that is flaking - the default-theme variation is fine. Let's skip it for now to unblock CI, and investigate it later.

File: frontend/src/metabase/visualizations/visualizations/LineChart/LineChart.stories.tsx
Comment (heypoom) [inline frontend/src/metabase/visualizations/visualizations/LineChart/LineChart.stories.tsx:56]: This story has been failing in the recent weeks, but I can't track down what's the cause yet. Let's skip it for the time being while we investigate.

**Review by heypoom — APPROVED** [review]
Checked that Loki tests are not failing anymore. I've also skipped the flaking tests on Line Charts for the time being. Thanks!

**Pull request merged**
